src/Messages/TSMessagesManager.h

```diff
@@ -20,14 +21,6 @@ NS_ASSUME_NONNULL_BEGIN
 @interface TSMessagesManager : NSObject
 
 - (instancetype)init NS_UNAVAILABLE;
-
-- (instancetype)initWithNetworkManager:(TSNetworkManager *)networkManager
```

Comment:
If we're going to leave these constructors for tests, perhaps we can wrap them in `#ifdef DEBUG` so that we don't accidentally use them in production code.